build_tools/docker/prod_digests.txt

```diff
@@ -6,6 +6,6 @@ gcr.io/iree-oss/frontends-nvidia@sha256:651e1c1c8dbe1cdfcca9648436e0c6fc9869d5df
 gcr.io/iree-oss/frontends-swiftshader@sha256:e70c6524cea980b9c6077a76e0ac90464c47eff4f27b6c74b8eaab3ff1fb35fc
 gcr.io/iree-oss/gradle-android@sha256:d36f3d172c3304e557cd3d9a13bae4590cb2a8d19d229b10f126423e7314a413
 gcr.io/iree-oss/riscv@sha256:24a401f9d47a16a0b460bf34a8485f087b7d88944ed661dd1201ce2d6c19df3e
-gcr.io/iree-oss/nvidia@sha256:b27c2feb5fc0e7125c5dd933c65c0f7029c157343c91269a17e892e76dd433a3
+gcr.io/iree-oss/nvidia@sha256:cad35c074dab074b6e36e2d5b4f36a2aba863d36013a93d50131db9bef66ec7f
```

Comment:
You likely need to also update the frontends-nvidia image as well

Reply:
I'm only enabling nvidia-cmake build. We don't have tf integration tests with cuda so I don't think we need to right now and this avoid creating big dockers

Reply:
Ok, sure